iothub_client/samples/iothub_client_device_cbor_sample/iothub_client_device_cbor_sample.c

```diff
@@ -0,0 +1,956 @@
+    //
+
+    // Set callback for connection status to IoT Hub.
+    rc = IoTHubDeviceClient_SetConnectionStatusCallback(iothub_client_handle, connection_status_callback, NULL);
+    if (rc != IOTHUB_CLIENT_OK)
+    {
+        (void)printf("Failed to set the connection status callback: return code %d.\n", rc);
+        exit(rc);
+    }
+
+    // Set callback for GET twin document request. - Approach #1 to requesting the twin document.
+    // Send asynchronous GET request for twin document. Connection will occur here.
+    rc = IoTHubDeviceClient_GetTwinAsync(iothub_client_handle, get_twin_async_callback, NULL);
+    if (rc != IOTHUB_CLIENT_OK)
+    {
+        (void)printf("Failed to send a GET request for the twin document asynchronously: return code %d.\n", rc);
+        exit(rc);
+    }
+
+    // Set callback for twin desired property PATCH messages from IoT Hub. - Approch #2 to requesting the twin document.
+    // Sends GET request for twin document as part of setting the callback.
+    rc = IoTHubDeviceClient_SetDeviceTwinCallback(iothub_client_handle, twin_desired_properties_callback, NULL);
+    if (rc != IOTHUB_CLIENT_OK)
+    {
+        (void)printf("Failed to set the twin document callback: return code %d.\n", rc);
+        exit(rc);
+    }
+
+    // Set callback for C2D messages from the service client application.
+    rc = IoTHubDeviceClient_SetMessageCallback(iothub_client_handle, receive_c2d_message_callback, NULL);
+    if (rc != IOTHUB_CLIENT_OK)
+    {
+        (void)printf("Failed to set the C2D message callback: return code %d.\n", rc);
+        exit(rc);
+    }
+
+    send_telemetry();
+
+    // Wait for desired property PATCH messages or C2D messages. Send Telemetry.
+    for (uint8_t message_count = 0; message_count < MAX_MESSAGE_COUNT; message_count++)
+    {
+        ThreadAPI_Sleep(TIMEOUT_RECEIVE_MS);
+        send_telemetry();
+    }
+}
+
+static void disconnect_device_client(void)
+{
+    IoTHubDeviceClient_Destroy(iothub_client_handle);
+    IoTHub_Deinit();
+}
+
+static void send_telemetry(void)
+{
+    int rc;
+
+    // Build the telemetry message in CBOR.
+    uint8_t telemetry_payload_buffer[TELEMETRY_MESSAGE_BUFFER_SIZE];
+    size_t telemetry_payload_length;
+    build_cbor_telemetry(
+        telemetry_payload_buffer, sizeof(telemetry_payload_buffer), &telemetry_payload_length);
+    message_handle = IoTHubMessage_CreateFromByteArray(telemetry_payload_buffer, telemetry_payload_length);
+
+    // Set the content type system property value for telemetry messages. This value is
+    // application-defined and must reflect what the service application expects.
+    rc = IoTHubMessage_SetContentTypeSystemProperty(message_handle, CONTENT_TYPE_TELEMETRY);
+    if (rc != IOTHUB_CLIENT_OK)
+    {
+        (void)printf("Failed to set message properties content type for telemetry: return code %d.\n", rc);
+        exit(rc);
+    }
+
+    (void)printf("\ntelemetry message payload: ");
```

Comment:
nit nit nit of nits: I don't know getting creative here :)
```suggestion
    (void)printf("\n-----telemetry message payload----- ");
```